docs/source/geopm::CPUActivityAgent.3.rst

```diff
@@ -4,7 +4,7 @@ geopm::CPUActivityAgent(3) -- agent for selecting CPU frequency based on CPU com
 Synopsis
 --------
 
-#include `<geopm/CPUActivityAgent.hpp> <https://github.com/geopm/geopm/blob/dev/libgeopm/src/CPUActivityAgent.hpp>`_
```

Comment:
Good change, but does beg the question of why we have a man page for a header that doesn't get installed...